CO/60737 - New Download Topic for Konvoy (#2659)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/latest/download/index.md

---
layout: layout.pug
navigationTitle: Download
title: Download Konvoy
menuWeight: 2
excerpt: Learn how to download an instance of Konvoy 
enterprise: false
---

When you are ready to download an instance of Konvoy, there are 2 possible scenarios:

1. You have already spoken to your sales representative and requested a download link. See <a href="#Downloading_Email">Downloading Konvoy via Email Link</a> for intructions. 
2. You want to try Konvoy and are interested in downloading a trial instance of the product. See  <a href="#Downloading_Email">Downloading Konvoy via the Trial Page</a> for instructions. 

## <a name="Downloading_Email">Downloading Konvoy via Email Link</a>

> After you contact your sales representative, an automated email from our license server will be sent to your account's primary contact with instructions on how to download Konvoy. Select the correct link within the email to start your download, then follow the instructions in the <a name=/ksphere/konvoy/latest/quick-start>Konvoy Quick Start</a> for your next steps.

Reviewer: ```suggestion
After you contact your sales representative, an automated email from our license server will be sent to your account's primary contact with instructions on how to download Konvoy. Select the link corresponding to the OS that you will be using within the email to start your download, then follow the instructions in the [Konvoy Quick Start](../quick-start/) for your next steps.
```
Konvoy has a link checker as part of our CI, `/ksphere/konvoy/latest/` is not correct in the context of that repo. This is also consistent with all the other links.

## <a name="Downloading_Trial>Downloading Konvoy via the Trial Page

> To request a free trial of Konvoy, go to https://d2iq.com/solutions/ksphere/konvoy#request-free-trial and fill out all required fields. After selecting **Submit**, an email will be sent to you with instructions on how to download Konvoy. Select the correct link within the email to start your download, then follow the instructions in the <a name=/ksphere/konvoy/latest/quick-start>Konvoy Quick Start</a> for your next steps.

Reviewer: ```suggestion
To request a free trial of Konvoy, go to https://d2iq.com/solutions/ksphere/konvoy#request-free-trial and fill out all required fields. After selecting **Submit**, an email will be sent to you with instructions on how to download Konvoy. Select the link corresponding to the OS that you will be using within the email to start your download, then follow the instructions in the [Konvoy Quick Start](../quick-start/) for your next steps.
```

---
layout: layout.pug
navigationTitle: Download
title: Download Konvoy
menuWeight: 2

> excerpt: Learn how to download an instance of Konvoy 

Reviewer 2: I suggest removing the words "an instance of"?  The download is for the Konvoy CLI that is used to install/deploy/manage Konvoy clusters.  Konvoy also refers to a distribution of kubernetes.

enterprise: false
---

When you are ready to download an instance of Konvoy, there are 2 possible scenarios:

1. You have already spoken to your sales representative and requested a download link. See <a href="#Downloading_Email">Downloading Konvoy via Email Link</a> for intructions. 
2. You want to try a trial version of Konvoy and are interested in downloading a trial instance of the product. See  <a href="#Downloading_Email">Downloading Konvoy via the Trial Page</a> for instructions. 

## <a name="Downloading_Email">Downloading Konvoy via Email Link</a>

> After you contact your sales representative, an automated email from our license server will be sent to your account's primary contact with instructions on how to download Konvoy. Select the correct link within the email to start your download. 

Reviewer 2: Replace the word "account" with "company".  I think account is ambiguous which could mean a login account or something - as opposed to the primary contact at their company - the person that acquired the license.
 
Might be better to have:
Licensed customers should have received an automated e-mail from the D2iQ entitlement server upon completion of a purchase.  The e-mail is sent to your company's primary support contact with instructions on how to download Konvoy.  If you have questions please contact your sales representative or send an e-mail to sales@d2iq.com.
